src/components/main-page/Mentors/config.tsx

```diff
@@ -0,0 +1,142 @@
+import { CardType } from "components/ui/Card/Card";
+
+export const mentorsData: CardType[] = [
+  {
+    name: "Natalia Sikora-Zimna",
+    company: "Nobl9",
+    role: "jury",
```

Comment:
alternatively, wouldn't it be better to always have an array and `elem.map(elem => t(elem)).join("/")`? It would remove the type check and we'll always treat the data the same way